deploy/esignet/install.sh

```
@@ -97,28 +259,254 @@ function installing_esignet() {
     PVC_CLAIM_NAME='esignet-pkcs12'
     ESIGNET_HELM_ARGS="--set persistence.enabled=true  \
-            --set volumePermissions.enabled=true \
-            --set persistence.size=$volume_size \
-            --set persistence.mountDir=\"$volume_mount_path\" \
-            --set persistence.pvc_claim_name=\"$PVC_CLAIM_NAME\"  \
-            --set extraEnvVarsCM={'esignet-global','config-server-share','artifactory-share'} \
-            --set extraEnvVarsAdditional.MOSIP_KERNEL_KEYMANAGER_HSM_KEYSTORE-TYPE=PKCS12 \
-            "
+                       --set volumePermissions.enabled=true \
+                       --set persistence.size=$volume_size \
+                       --set persistence.mountDir=\"$volume_mount_path\" \
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)